krux_troposphere/troposphere.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+#
+# © 2016 Krux Digital, Inc.
+#
+
+#
+# Standard libraries
+#
+
+from __future__ import absolute_import
+from abc import ABCMeta, abstractmethod
+import uuid
+
+#
+# Third party libraries
+#
+
+import troposphere
+import botocore.exceptions
+
+#
+# Internal libraries
+#
+
+from krux.logging import get_logger
+from krux.stats import get_stats
+from krux.cli import get_parser, get_group
+from krux_boto.boto import Boto3, add_boto_cli_arguments
+
+
+NAME = 'krux-troposphere'
+
+
+def get_troposphere(args=None, logger=None, stats=None):
+    """
+    Return a usable Troposphere object without creating a class around it.
+
+    In the context of a krux.cli (or similar) interface the 'args', 'logger'
+    and 'stats' objects should already be present. If you don't have them,
+    however, we'll attempt to provide usable ones for the Troposphere setup.
+
+    (If you omit the add_troposphere_cli_arguments() call during other cli setup,
+    the Boto object will still work, but its cli options won't show up in
+    --help output)
+
+    (This also handles instantiating a Boto3 object on its own.)
+    """
+    if not args:
+        parser = get_parser()
+        add_troposphere_cli_arguments(parser)
+        args = parser.parse_args()
+
+    if not logger:
+        logger = get_logger(name=NAME)
+
+    if not stats:
+        stats = get_stats(prefix=NAME)
+
+    boto = Boto3(
+        log_level=args.boto_log_level,
+        access_key=args.boto_access_key,
+        secret_key=args.boto_secret_key,
+        region=args.boto_region,
+        logger=logger,
+        stats=stats,
+    )
+    return Troposphere(
+        boto=boto,
+        logger=logger,
+        stats=stats,
+    )
+
+
+def add_troposphere_cli_arguments(parser, include_boto_arguments=True):
+    """
+    Utility function for adding Troposphere specific CLI arguments.
+    """
+    if include_boto_arguments:
```

Comment:
Can you elaborate on "that problem"? I do not know which problem you are referring to.